src/views/Namespace/List.vue

```diff
@@ -109,6 +109,12 @@ export default {
   data () {
     return {
       query: '',
+      myItems: ['pl'],
```

Comment:
What does this do?, same for variable below?

Reply:
was forgotten; removing